pkgs/tools/networking/networkmanager/dmenu/default.nix

```diff
@@ -4,22 +4,25 @@
 let inherit (python3Packages) python pygobject3;
 in stdenv.mkDerivation rec {
   pname = "networkmanager_dmenu";
-  version = "1.6.0";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
     owner = "firecat53";
     repo = "networkmanager-dmenu";
-    rev = version;
-    sha256 = "1liidqh8c33pxyb07qyj0jkd0fdak73g9r2iwiq62vfzrpik09k0";
+    rev = "v${version}";
+    sha256 = "sha256-2IDe8TISxfO8TtxDu8rffU1jQChHkbgNO3Sk6DJRJZM=";
   };
 
   buildInputs = [ glib python pygobject3 gobject-introspection networkmanager python3Packages.wrapPython ];
 
   dontBuild = true;
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/share/applications
     cp networkmanager_dmenu $out/bin/
+    cp networkmanager_dmenu.desktop $out/share/applications
+    cp README.md $out/share
+    cp config.ini.example $out/share
```

Comment:
These should probably go to a subdirectory like `doc/networkmanager_dmenu` to prevent conflicts.

Reply:
Done.
I also upgraded to the newest version.